queries/water-z9.pgsql

```diff
@@ -35,6 +35,24 @@ FROM
         AND way_area::bigint > 102400 -- 4px
         AND mz_way12 && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
+           waterway AS kind,
+           'openstreetmap.org' AS source,
+           way AS __geometry__,
+           mz_id AS __id__
+
+    FROM planet_osm_line
+
+    WHERE
+        waterway IN ('canal', 'river')
```

Comment:
For the other queries, we typically have a filter on the area. The idea is that at the lower zoom levels, like perhaps this one, we wouldn't show waterways unless they were of a certain size. I'm not sure if that makes sense to do here, as it could be that the tag value itself would already provide the sufficient filtering, just pointing it out for consideration.